build/dsc.rake

```diff
@@ -115,6 +116,15 @@ eod
       m.target_module_path = module_path
       msgs = m.build_dsc_types
       msgs.each{|m| puts "#{m}"}
+      dsc_types = m.get_dsc_types
```

Comment:
@Iristyle oops. Just noticed this one. I can clean it up, it's removing a single line.